source/utilities.cc

```diff
@@ -221,15 +226,15 @@ namespace aspect
                                  + "The current structure vector has " + std::to_string(n_values_per_key->size()) + " entries, but there are "
                                  + std::to_string(n_fields) + " field names." ));
 
-        for (const auto &entry: parsed_map)
+        for (const std::pair<std::string, double> &key_and_value: parsed_map)
```

Comment:
I really like this change. I always struggle with pairs because from the name it's usually not clear what pair.first and pair.second are. This makes it really easy to read.